src/popup/components/Modals/QrCodeReader.vue

```diff
@@ -217,14 +209,11 @@ export default defineComponent({
         return;
       }
 
+      isCameraReady.value = true;
```

Comment:
Now as we don't need to wait for initializing the QR code reader I think we can refactor the code below in a following way:
```
props.resolve((IS_MOBILE_APP) ? await scanMobile() : await scanWeb());
```
Also the TODO comment can be removed.